controllers/userController.js

```diff
@@ -0,0 +1,20 @@
+const userService = require('../services/userService');
+const { catchAsync } = require('../utils/error');
+
+const kakaoLogin = catchAsync(async (req, res) => {
+  const kakaoToken = req.headers.authorization;
+  if (!kakaoToken) {
+    const err = new Error('missingAuthCode');
+    err.statusCode = 401;
+    throw err;
+  }
+
+  const result = await userService.kakaoLogin(kakaoToken);
+  const code = result.statusCode;
+  const accessToken = result.accessToken;
+  res.status(code).json({ accessToken });
```

Comment:
잘 반영해주셨네요!! 👍 다만 아래와 같이 구조분해 할당을 활용하면 코드 라인을 줄일 수 있을 것 같네요

```suggestion
  const { statusCode, accessToken } = await userService.kakaoLogin(kakaoToken);
  
  res.status(statusCode).json({ accessToken });
```